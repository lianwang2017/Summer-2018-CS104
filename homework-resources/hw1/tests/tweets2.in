@rt, aren't we looking into #dropbox for our files

No we are not @dpg, #GoogleDrive

@rt :: not sure!
