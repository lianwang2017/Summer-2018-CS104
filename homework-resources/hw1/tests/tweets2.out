1. Number of tweets=3
2. Unique users
rt,
dpg,
rt
3. Unique hashtags
dropbox
GoogleDrive
